cms/templates/js/content-group-details.underscore

```diff
@@ -32,7 +32,7 @@
             </li>
         <% } else { %>
             <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Cannot delete when in use by a unit') %>">
-                <button class="delete action-icon is-disabled" aria-disabled="true" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+                <button class="delete action-icon is-disabled" disabled="true" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
```

Comment:
@clrux I will create a bug for this issue and we can fix it at the time that Studio issues are addressed. I'm not sure what the best pattern for fixing this sort of issue is-- honestly, I'd be tempted to just always show the buttons, but that sort of decision should be made with UX input.

For now, I really need to get @ssemenova back to fixing LMS a11y issues.

I created the bug and put it in my team's backlog-- https://openedx.atlassian.net/browse/TNL-4850